enclave_wrangler/models.py

```diff
@@ -281,7 +281,9 @@ def add_mappings(csv_str: str):
        includeMapped,                includeMapped
        isExcluded,                   isExcluded
        createdBy,                    created_by
-       createdAt,                    created_at""")
+       createdAt,                    created_at
+       annotation,                   annotation
+       sourceApplication,            source_application""")
```

Comment:
## Bug fix: missing `annotation` and `sourceApplication`
These weren't getting imported for `concept_set_version_item` because they were missing from this mapping table.